kubernetes/kubernetes_service.go

```diff
@@ -22,6 +22,21 @@ import (
 // GetNamespace fetches and returns the specified namespace definition
 // from the cluster
 func (in *IstioClient) GetNamespace(namespace string) (*core_v1.Namespace, error) {
+	if in.k8sCache != nil {
+		nss, err := in.GetNamespaces("")
```

Comment:
yeah, this operation is clusterwide in k8s, in openshift projects make the filter per user "visibiliy".